src/components/BadUserList.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
When you add new code, it is super important to also add new tests (or really, the ideal way of working is to add automated tests first and then write the code, a technique known as Test Driven Development).

The goal with tests is to enforce all of the little details and edge cases so that someone does not need to understand all of the details of your implementation if they want to modify it. They can run your tests and know, if one of them breaks, that they might have accidentally broken the code. For a small program like Book a Desk, this might seem trivial, but Book a Desk will grow. At a certain size, it becomes impossible to know if you broke something unless a test is there to catch it for you.

So here are the tests that I think you should be adding:

`Given a date and a user list When displaying the user list Then the correct date is shown in the heading`

`Given a list of bookings and a user list When displaying the user list Then the users are shown`
( this test is specifically about having 1 item per booking)

`Given a booking When displaying the booking Then the username is shown`
(this test is for the BadUserListItem, where we make sure the username is shown)